app/models/user.rb

```diff
@@ -421,6 +421,16 @@ def relationships
     @relationships ||= get_relationships_array
   end
 
+  def identity_serial
+    {
+      'uuid' => uuid,
+      'first_name' => first_name,
+      'last_name' => last_name,
+      'email' => email,
+      'account_id' => account.id
```

Comment:
Please don't use `Account` model for new functionality, we're deprecating this. `user.user_account.id` is the appropriate alternative to use (and you can grab the ICN off of that).